### ISSUE TITLE
Liquid Warning: Liquid syntax error

### ISSUE THREAD

**Issue by NeverMin** [Phlow/feeling-responsive#145]
Liquid Warning: Liquid syntax error (line 7): Expected end_of_string but found id in "{{ post.title escape_once }}"

**Comment by ghost** [comment]
@NeverMin please add your `jekyll -v && git log -1`

**Comment by odit** [comment]
Here we go:
```
never@never-pc:/Project/feeling-responsive$  rvm use 2.2.4
Using /home/never/.rvm/gems/ruby-2.2.4
never@never-pc:/Project/feeling-responsive$  jekyll -v
jekyll 3.3.1
never@never-pc:/Project/feeling-responsive$  git log -1
commit bf774279029db1a9247977a36327d45afe215fdb
Author: NeverMin <never.min@gmail.com>
Date:   Fri Apr 21 15:11:39 2017 +0800

    fixed Liquid syntax error #145
never@never-pc:/Project/feeling-responsive$ 
```

**Comment by NeverMin** [comment]
Here we go:
```
never@never-pc:/Project/feeling-responsive$  rvm use 2.2.4
Using /home/never/.rvm/gems/ruby-2.2.4
never@never-pc:/Project/feeling-responsive$  jekyll -v
jekyll 3.3.1
never@never-pc:/Project/feeling-responsive$  git log -1
commit bf774279029db1a9247977a36327d45afe215fdb
Author: NeverMin <never.min@gmail.com>
Date:   Fri Apr 21 15:11:39 2017 +0800

    fixed Liquid syntax error #145
never@never-pc:/Project/feeling-responsive$ 
```

**Comment by TWiStErRob** [comment]
This should be fixed by #146.

(@NeverMin GitHub doesn't parse the meaning of the commit message, it simply searches for "fixed #NNN", you can't put stuff in the middle of it: https://help.github.com/articles/closing-issues-via-commit-messages/)

**Comment by NeverMin** [comment]
@TWiStErRob :+1: Copy, Thank you for your TIPS.
